code/datums/components/customizable_reagent_holder.dm

```diff
@@ -92,18 +95,17 @@
 
 	var/atom/atom_parent = parent
 	var/ingredients_listed = ""
-	if (LAZYLEN(ingredients))
-		for (var/i in 1 to ingredients.len)
-			var/obj/item/ingredient = ingredients[i]
-			var/ending = ", "
-			switch(length(ingredients))
-				if (2)
-					if (i == 1)
-						ending = " and "
-				if (3 to INFINITY)
-					if (i == ingredients.len - 1)
-						ending = ", and "
-			ingredients_listed += "\a [ingredient.name][ending]"
+	for (var/i in 1 to LAZYLEN(ingredients))
+		var/obj/item/ingredient = ingredients[i]
+		var/ending = ", "
+		switch(length(ingredients))
+			if (2)
+				if (i == 1)
+					ending = " and "
+			if (3 to INFINITY)
+				if (i == LAZYLEN(ingredients) - 1)
+					ending = ", and "
+		ingredients_listed += "\a [ingredient.name][ending]"
```

Comment:
this code makes me very depressed (this isn't hot right? would it be useful to handle as a list instead?

Reply:
I actually have no idea why this doesn't just use english_list

Reply:
the \a maybe?